haskell-indexer-translate/src/Language/Haskell/Indexer/Translate.hs

```diff
@@ -238,6 +240,11 @@ data Relation = Relation
 data RelationKind = ImplementsMethod | InstantiatesClass
     deriving (Eq, Ord, Show)
 
+-- | Module import
+data Import = Import
```

Comment:
Can we just use the `ModuleTick` instead? It would contain the pkg+module, as well as the Span (of the import here).